src/app/@ansyn/ansyn/modules/menu-items/results/components/results-table/results-table.component.less

```diff
@@ -174,55 +185,15 @@
 						text-overflow: ellipsis;
 					}
 
-					div.results-table-case-name-col {
+					div.results-table-date-col {
+						flex: 2;
 						overflow: hidden;
 						text-overflow: ellipsis;
-						min-width: @date-width;
 					}
 
 					.results-table-auto-save-col {
 						padding-top: 20px;
-					}
-				}
-			}
-		}
-	}
-
-	&.rtl {
```

Comment:
RTL code missing? Please add it.

**Please check the new design in both LTR and RTL modes** 
(they are switched according to "direction" field in translations in config)